docs/content/querying/filters.md

```diff
@@ -18,6 +18,18 @@ This is the equivalent of `WHERE <dimension_string> = '<dimension_value_string>'
 
 The selector filter supports the use of extraction functions, see [Filtering with Extraction Functions](#filtering-with-extraction-functions) for details.
 
+### Column Comparison filter ###
```

Comment:
I think the trailing `###` is ignored, so this is probably harmless, but it's also not necessary.